disnake/invite.py

```diff
@@ -95,17 +101,21 @@ class PartialInviteChannel:
         The partial channel's type.
     """
 
-    __slots__ = ("id", "name", "type")
+    __slots__ = ("id", "name", "type", "_recipients")
 
     def __init__(self, data: InviteChannelPayload):
```

Comment:
Could you update the payload type accordingly? Perhaps a new type
```py
class InviteChannel(PartialChannel, total=False):
    name: Optional[str]
    recipients: List[<...>]
```
There's also an `icon` field that's currently unsupported, could add that here or in another PR

Reply:
I don't think icon is documented?

Reply:
it is:
![image](https://user-images.githubusercontent.com/8530778/170144441-7ca2cc9c-a576-410e-a5f0-5a4308ff9325.png)
https://discord.com/developers/docs/resources/channel#channel-object-channel-structure

Reply:
https://github.com/DisnakeDev/disnake/pull/498/commits/a2ef1f7931c19cdc5fd000e3df1c8d9eff3b7144